packages/cli/src/cli/autoformat.rs

```diff
@@ -147,11 +149,13 @@ async fn format_file(
 ///
 /// Runs using Tokio for multithreading, so it should be really really fast
 ///
-/// Doesn't do mod-descending, so it will still try to format unreachable files. TODO.
-async fn autoformat_project(check: bool, split_line_attributes: bool) -> Result<()> {
-    let crate_config = dioxus_cli_config::CrateConfig::new(None)?;
-
-    let files_to_format = get_project_files(&crate_config);
+/// Doesnhttps://www.rustwiki.org.cn/zh-CN/std/'t do mod-descending, so it will still try to format unreachable files. TODO.
```

Comment:
Is the link to https://www.rustwiki.org.cn/zh-CN/std/ intentional?

Reply:
no, by mistake